components/NewPersonView/Event.tsx

```diff
@@ -33,21 +37,20 @@ const Event = ({ event }: Props): React.ReactElement => {
       }`}
     >
       <h3 className="lbh-heading-h3">
-        <CaseLink
-          formName={event.formName}
-          externalUrl={event.caseFormUrl}
-          caseFormData={event.caseFormData}
-          recordId={event.recordId}
-          personId={event.personId}
-        >
-          {displayName}
-        </CaseLink>
```

Comment:
Do we even need the CaseLink component now, it only exists in the <CaseTable> component which doesn't seem to be used anywhere?

Reply:
it's still used in a few places - i am keen to deprecate it